src/qibolab/sweeper.py

```diff
@@ -1,20 +1,65 @@
 from dataclasses import dataclass
+from enum import Enum, auto
 from typing import Optional
 
 import numpy.typing as npt
 
+from qibolab.pulses import PulseType
+
+
+class Parameter(Enum):
+    """Enum type for sweeping parameter."""
```

Comment:
```suggestion
    """Sweeping parameters."""
```

The first part is redundant: it is written above that is an `Enum` :)